### ISSUE TITLE
C++/WinRT Runtime Component doesn't notice missing merged winmd file

### ISSUE THREAD

**Issue by FrankHeimes** [microsoft/cppwinrt#827]
1. Build the attached sample project [RuntimeComponent1.zip](https://github.com/microsoft/cppwinrt/files/5742246/RuntimeComponent1.zip), which was created by the Visual Studio project wizard.
2. Build the project again.
3. Nothing is built because all outputs of the project are up-to-date.
4. Delete the file `Debug\Merged\RuntimeComponent1.winmd`
5. Build the project again.
3. Still nothing is built even though `RuntimeComponent1.winmd` is missing.

This is a problem because dependent projects that consume `RuntimeComponent1.winmd` fail to build until `RuntimeComponent1` is forcibly rebuilt, recreating `RuntimeComponent1.winmd` in the process.

**Comment by FrankHeimes** [comment]
I just noticed that Chrome considers the archive `RuntimeComponent1.zip` to be dangerous, but rest assured that it contains only sources generated by Visual Studio. Internet Explorer is less wary.

**Comment by jlaanstra** [comment]
The file in the merged folder is an intermediate file and is therefore not automatically generated as it is not the output of any target and shouldn't be consumed by other projects. The actual output winmd file is in the bin folder.

**Comment by FrankHeimes** [comment]
@jlaanstra 

The build creates exactly these `.winmd` files:
```
RuntimeComponent1\RuntimeComponent1\x64\Debug\Unmerged\Class.winmd
RuntimeComponent1\RuntimeComponent1\x64\Debug\Merged\RuntimeComponent1.winmd
RuntimeComponent1\x64\Debug\RuntimeComponent1\RuntimeComponent1.winmd
RuntimeComponent1\x64\Debug\RuntimeComponent1\RuntimeComponent1.dll
```
I assume, the folder holding `RuntimeComponent1.dll` is what you referred to as the _bin_ folder.
So I deleted `RuntimeComponent1\x64\Debug\RuntimeComponent1\RuntimeComponent1.winmd` to verify your statement.
By running `Build solution`, that file was actually restored.
Then I deleted `RuntimeComponent1\x64\Debug\RuntimeComponent1\RuntimeComponent1.winmd` _again_ and ran `Build solution` _again_.
But now the project remains current even though the missing `.winmd` file is never restored.

I double checked that again. The project restores a missing  `.winmd` file in the output folder only _once_ but never again thereafter.
